backend/src/main/java/com/festago/fcm/application/FCMNotificationEventListener.java

```diff
@@ -0,0 +1,90 @@
+package com.festago.fcm.application;
+
+import com.festago.common.exception.ErrorCode;
+import com.festago.common.exception.InternalServerException;
+import com.festago.entry.dto.event.EntryProcessEvent;
+import com.festago.fcm.domain.FCMChannel;
+import com.festago.fcm.dto.MemberFCMResponse;
+import com.google.firebase.messaging.AndroidConfig;
+import com.google.firebase.messaging.AndroidNotification;
+import com.google.firebase.messaging.BatchResponse;
+import com.google.firebase.messaging.FirebaseMessaging;
+import com.google.firebase.messaging.FirebaseMessagingException;
+import com.google.firebase.messaging.Message;
+import com.google.firebase.messaging.SendResponse;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Profile;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.event.TransactionPhase;
+import org.springframework.transaction.event.TransactionalEventListener;
+
+@Component
+@Profile({"dev", "prod"})
+public class FCMNotificationEventListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(FCMNotificationEventListener.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(FCMNotificationEventListener.class);
```

아래 코드와 통일하면 좋을 것 같아요!